index.html

```diff
@@ -44,6 +44,217 @@ <h3>I'am a Software Developer</h3>
+          sign-ups required. has been the 
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+
+      <section class="card2">
+        <h3>Professional Art <br> Printing Data</h3>
+      
+        <p>A daily selection of privately 
+          personalized reads; no accounts or 
+          sign-ups required. has been the 
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+
+      <section class="card2">
+        <h3>Professional Art <br> Printing Data</h3>
+      
+        <p>A daily selection of privately 
+          personalized reads; no accounts or 
+          sign-ups required. has been the 
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+
+      <section class="card2">
+        <h3>Professional Art <br> Printing Data</h3>
+      
+        <p>A daily selection of privately 
+          personalized reads; no accounts or 
+          sign-ups required. has been the 
+          industry's standard
+        </p>
+        
+        <ul class="buttons1">
+          <li><button class="button1">html</button></li>
+          <li><button class="button1">bootstrap</button></li>
+          <li><button class="button1">Ruby</button></li>
+        </ul>
+        <br>
+        <div>
+          <button class="orange-button orange-button2">See Project</button>
+        </div>
+      </section>
+    </section>
+
+    <section class="container2">
+      <section class="aboutMe">
+        <h1>About me</h1>
+
+        <p>
+          Hello I’m a software developer! I can help 
+          you build a product , feature or website 
+          Look through some of my work and 
+          experience! If you like what you see and 
+          have a project you need coded, don’t 
+          hestiate to contact me.
+        </p>
+
+        <div>
+          <button class="orange-button orange-button3">Get My Resume</button>
+        </div>
+      </section>
+
+      <hr>

```

Comment:
- [ ] You must stick to the design as much as possible (e.g. font, colours, images,, margins) using the templates in Figma.
- Use the appropriate font for this section,
- Centralise this section too,
- Reduce the width of the horizontal line and centralize it.
_See the images below for clarity._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/121535231-d46bd680-c9b6-11eb-80e2-06ed03452635.png) | ![image](https://user-images.githubusercontent.com/49621279/121535315-e77ea680-c9b6-11eb-8f63-a764245129cf.png)